lib/bls.ex

```diff
@@ -54,10 +54,6 @@ defmodule Bls do
     :erlang.nif_error(:nif_not_loaded)
   end
 
-  @spec derive_pubkey(privkey()) :: {:ok, pubkey()} | {:error, any()}
-  def derive_pubkey(_private_key) do
-    :erlang.nif_error(:nif_not_loaded)
```

Comment:
Let's keep this one instead of the other. That way if the NIF is not loaded (and the implementation is not overwritten), calling the function raises a clear error.